### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by sigee**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,7 @@
 Contributing to Apache Commons CSV
 ======================
 
-You have found a bug or you have an idea for a cool new feature? Contributing code is a great way to give something back to
+You have found a bug, or you have an idea for a cool new feature? Contributing code is a great way to give something back to
```
Comment (aherbert) [inline CONTRIBUTING.md:44]: This is auto generated text so this is not the place to fix this. Can you revert please.

File: pom.xml [hunk 2]
```diff
@@ -404,6 +404,13 @@
       </plugin>
     </plugins>
   </reporting>
+  <repositories>
```
Comment (aherbert) [inline pom.xml:407]: This repository is already included from the parent pom: org.apache:apache

File: pom.xml [hunk 3]
```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>4.11.0</version>
+      <version>5.11.0</version>
```
Comment (aherbert) [inline pom.xml:47]: I am not sure why you are upgrading this dependency; the pr has no use of this plugin.

**Review by aherbert — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/commons/csv/Lexer.java [hunk 4]
```diff
@@ -22,6 +22,8 @@
 import java.io.Closeable;
 import java.io.IOException;
 
+import org.apache.commons.io.IOUtils;
```
Comment (aherbert) [inline src/main/java/org/apache/commons/csv/Lexer.java:25]: We do not require this import for the code. You can use it directly in the javadoc.

File: src/main/java/org/apache/commons/csv/Lexer.java [hunk 5]
```diff
@@ -467,7 +469,7 @@ boolean readEndOfLine(int ch) throws IOException {
      * On return, the next character is available by calling {@link ExtendedBufferedReader#getLastChar()}
      * on the input stream.
      *
-     * @return the unescaped character (as an int) or {@link Constants#EOF} if char following the escape is
+     * @return the unescaped character (as an int) or {@link IOUtils#EOF} if char following the escape is
```
Comment (aherbert) [inline src/main/java/org/apache/commons/csv/Lexer.java:472]: ```{@link org.apache.commons.io.IOUtils#EOF IOUtils.EOF}```

**Review by aherbert — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (garydgregory) [reply pom.xml]: Worse, it's not compatible with Java 8 🙁

**Comment by garydgregory** [comment]
Closing, see comments.

**Pull request closed without merging**